Backend/app/rider-platform/rider-app/Main/src/API/Dashboard/RideBooking/Booking.hs

```diff
@@ -69,4 +70,9 @@ callBookingStatus merchantId bookingId personId = do
 callBookingList :: ShortId DM.Merchant -> Id DP.Person -> Maybe Integer -> Maybe Integer -> Maybe Bool -> Maybe SRB.BookingStatus -> FlowHandler DBooking.BookingListRes
 callBookingList merchantId personId mbLimit mbOffset mbOnlyActive bkngStatus = do
   m <- withFlowHandlerAPI $ findMerchantByShortId merchantId
-  UB.bookingList (personId, m.id) mbLimit mbOffset mbOnlyActive bkngStatus
+  res <- UB.bookingList (personId, m.id) mbLimit mbOffset mbOnlyActive bkngStatus
```

Comment:
Why is this added at booking list side ?

Reply:
The dashboard side is internally hitting Booking list API. That's why added here

Reply:
done with the changes